dev/Detours/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Build.Tasks.Git" version="1.1.0-beta-20204-02" targetFramework="native" developmentDependency="true" />
```

Comment:
>1.1.0-beta-20204-02 [](http://example.com/codeflow?start=51&length=19)

i hope one day we can have those versions defined in some common properties, therefore everyone is on the same page, and update only requires changing one place.

Reply:
Definitely something to look at in the future - also common .props file that projects reference so we don't have so many edits.

Reply:
Nice idea but alas, packages.config has no #include/import/etc mechanism :-(